.gitignore

```diff
@@ -1,2 +1,3 @@
 *.egg-info/
 *.pyc
+venv
```

Comment:
I name my virtualenvs `venv` and `venv3` respectively. Although this might not be a problem for much longer with the imminent move to py3 can we have the `venv*` pattern here like in https://github.com/alphagov/digitalmarketplace-api/blob/master/.gitignore ?